src/main/java/plugily/projects/buildbattle/commands/arguments/game/GuessArgument.java

```diff
@@ -50,7 +50,7 @@ public void execute(CommandSender sender, String[] args) {
         Player player = (Player) sender;
         BaseArena arena = (BaseArena) registry.getPlugin().getArenaRegistry().getArena(player);
 
-        if(!(arena instanceof GuessArena) || arena.getArenaState() != ArenaState.IN_GAME) {
+        if(arena.getArenaState() != ArenaState.IN_GAME || !(arena instanceof GuessArena)) {
```

Comment:
now it would need null check? reverting seems best option for me on that

Reply:
I did not consider the null check, but the effect of `instanceof` on the CPU. An enum comparison is faster than an `instanceof` check.

Reply:
but would require another null check before, so I guess instanceof calling is faster than null and after that again, it saves us one call if we do it in first place

Reply:
Oh, actually you're right, I forgot that the above `getArena` can return null if the current player is no playing. So I am gonna revert this change.